### PR TITLE
add command for reqtimeout

### PR DESCRIPTION
**Pull request by voroyam**
added a command that disables the module that prevents big file uploads in ubuntu

### CODE REVIEW

File: modules/admin_manual/pages/configuration/files/big_file_upload_configuration.adoc [hunk 1]
```diff
@@ -57,6 +57,18 @@ using this module and getting failed uploads of large files either
 disable it in your Apache config or raise the configured
 `RequestReadTimeout` timeouts.
 
+On Ubuntu you can diable this mod by running this command:
```
Comment (phil-davis) [inline modules/admin_manual/pages/configuration/files/big_file_upload_configuration.adoc]: s/diable/disable

File: modules/admin_manual/pages/configuration/files/big_file_upload_configuration.adoc [hunk 2]
```diff
@@ -57,6 +57,18 @@ using this module and getting failed uploads of large files either
...
+
+....
+a2dismod reqtimeout
+....
+
+On CentOS you have to edit the `/etc/httpd/conf/httpd.conf`.
```
Comment (settermjd) [inline modules/admin_manual/pages/configuration/files/big_file_upload_configuration.adoc]: In this sentence, "the" is not required.

**Comment by voroyam** [comment]
> @voroyam, I've added a few, additional changes. If you're happy with them, I'm happy to approve and merge it.

yes, this is better than my quick copy paste. Thanks :)

**Comment by settermjd** [comment]
> > @voroyam, I've added a few, additional changes. If you're happy with them, I'm happy to approve and merge it.
> 
> yes, this is better than my quick copy paste. Thanks :)

Happy to help.

**Pull request merged**